docs/getting-started/quick-start.md

```diff
@@ -37,21 +37,22 @@ namespace.
 
 ## Deploying the operator
 
-To deploy the operator simply run
+To deploy the operator in the current namespace simply run
 
-```$xslt
-$ make deploy
+```shell
+cd deploy
+./install_opr.sh
```

Comment:
how do you check whether this worked?

I misread the current text and tried ` kubectl get pod -n activemq-artemis-operator` but that's only applicable if you're doing a cluster wide deploy